src/std/text.ab

```diff
@@ -11,11 +11,25 @@ pub fun replace(source, pattern, replacement) {
 /// Replaces all occurences of a regex pattern in the content with provided replacement text
 ///
 /// Function uses `sed`
-pub fun replace_regex(source: Text, pattern: Text, replacement: Text): Text {
-    return unsafe $echo "{source}" | sed -e "s/{pattern}/{replacement}/g"$
+pub fun replace_regex(source: Text, pattern: Text, replacement: Text, extended: Bool = false): Text {
+    unsafe {
+        if extended {
+            // GNU sed versions 4.0 through 4.2 support extended regex syntax,
+            // but only via the "-r" option; use that if the version information
+            // contains "GNU sed".
+            unsafe $re='\bGNU sed\b'; [[ \$(sed --version 2>/dev/null) =~ \$re ]]$
```

Comment:
It has been suggested that I search for "Copyright" and "Free Software Foundation" on the same line, but searching "GNU sed" should not give us false positives, and should run slightly faster.

Reply:
you are doing a micro optimization here. not even to mention that busybox sed would match that:

```bash
if echo $version | grep 'This is not GNU sed version ' 2>&1 > /dev/null; then
    echo is alpine (busybox)
fi
```

Reply:
Fair enough. I've changed it to match `\bCopyright\b.+\bFree Software Foundation\b`; does this resolve your request?